### ISSUE TITLE
questions not found

### ISSUE THREAD

**Issue by rcannood** [dynverse/dyno#21]
_Originally posted by @ysu2015 in https://github.com/dynverse/dyno/issues/9#issuecomment-422484005_

Hi @zouter,

when I run the commands in the tutorial, I got another error.
```r
library(dyno)
library(tidyverse)
data("fibroblast_reprogramming_treutlein")
dim(fibroblast_reprogramming_treutlein$expression)
task <- wrap_expression(
  counts = fibroblast_reprogramming_treutlein$counts,
  expression = fibroblast_reprogramming_treutlein$expression
)
guidelines <- guidelines_shiny(task)

Error in setdiff(names(questions), given_question_ids) : 
  object 'questions' not found
```

**Comment by rcannood** [comment]
Hello @ysu2015,

Your problem should have been fixed in #20. Did you try reinstalling dynguidelines (`devtools::install_github("dynverse/dynguidelines")`) and see whether the problem still persists?

Thanks!
Robrecht

**Comment by rcannood** [comment]
Sorry, @zouter just let me know that this issue was already handled in #20. Closing this issue.
